### PR TITLE
AMPA 367 Implement Video Player UI, intergrate play, pause, forward, backward

### PR DESCRIPTION
**Pull request by nghiep9523**
- Done styling controller and 3-video layout.

- Play, pause, forward, backward can be used now

- Layout setting popup implemented

### CODE REVIEW

File: src/containers/path/path.js [hunk 1]
```diff
@@ -20,7 +21,7 @@ class PathPage extends Component {
   render() {
     return (
       <div className="path-page">
-        path page
+        <VideoPlayer />
```
Comment (nhatdev1193) [inline src/containers/path/path.js]: cai ``` < VideoPlayer /> ``` minh se implement trong sylabus page nen @nghiep9523 co gi e xoa dong nay di nha  😄  neu de demo thi nen xoa truoc khi commit 😄

**Review by nhatdev1193 — CHANGES_REQUESTED** [review]

File: src/containers/path/path.js (hunk 1 above)
Comment (nghiep9523) [reply src/containers/path/path.js]: oke anh :smile:

File: src/components/shared/video_player.js [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    }
+
+    handleSettingClick(e) {
+      e.preventDefault();
+      this.setState({settingOpened: !this.state.settingOpened});
+    }
+
+    openSpeedControl(e) {
+      e.preventDefault();
+      this.setState({speedOpened: !this.state.speedOpened});
```
Comment (nhatdev1193) [inline src/components/shared/video_player.js]: them set state cho cai setting layout, de no tat luc openSpeedControl va nguoc lai.

File: src/containers/path/path.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React, {Component} from 'react';
 import './path.style.css';
+import VideoPlayer from '../../components/shared/video_player';
```
Comment (nhatdev1193) [inline src/containers/path/path.js]: xoa cai nay di nha e

**Review by nhatdev1193 — CHANGES_REQUESTED** [review]

File: src/components/shared/video_player.js [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+          volume: 100
+        }
+
+        this.placeHolderImage = [
+          'http://via.placeholder.com/215x105',
+          'http://via.placeholder.com/215x105',
+          'http://via.placeholder.com/215x105',
+          'http://via.placeholder.com/215x105',
+          'http://via.placeholder.com/215x105',
+          'http://via.placeholder.com/215x105',
```
Comment (nhatdev1193) [inline src/components/shared/video_player.js]: e contact vs Minh ben Design de xin may cai hinh cua layout roi edit cai nay lai.

**Review by nhatdev1193 — CHANGES_REQUESTED** [review]

**Review by nhatdev1193 — APPROVED** [review]

**Review by votheduy154 — APPROVED** [review]

**Pull request merged**